# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1706535557 -3600
# Parent  195d778f35a145155a003a673166786b4807dbd2
NNN10 - Reduce more console noise, ext-tabs.js:75.

diff --git a/mail/components/extensions/parent/ext-tabs.js b/mail/components/extensions/parent/ext-tabs.js
--- a/mail/components/extensions/parent/ext-tabs.js
+++ b/mail/components/extensions/parent/ext-tabs.js
@@ -67,16 +67,17 @@ let tabListener = {
    */
   awaitTabReady(nativeTabInfo) {
     let deferred = this.tabReadyPromises.get(nativeTabInfo);
     if (!deferred) {
       deferred = PromiseUtils.defer();
       let browser = getTabBrowser(nativeTabInfo);
       if (
         !this.initializingTabs.has(nativeTabInfo) &&
+        browser &&
         (browser.innerWindowID ||
           ["about:blank", "about:blank?compose"].includes(
             browser.currentURI.spec
           ))
       ) {
         deferred.resolve(nativeTabInfo);
       } else {
         this.initTabReady();
